src/styles/index.css

```diff
@@ -0,0 +1,124 @@
+body {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  align-items: center;
+  justify-content: center;
+}
+
+.product-card__image {
+  background-image: url('../images/imac.jpeg');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
```

Comment:
The 'margin' shorthand is specified with pixel values, which is not very flexible. Consider using relative units like 'em' or 'rem' for margins to make the design more responsive and accessible.